crates/bevy_pbr/src/render/light.rs

```diff
@@ -865,6 +865,7 @@ pub fn prepare_lights(
         gpu_directional_lights[index] = GpuDirectionalLight {
             // Filled in later.
             cascades: [GpuDirectionalCascade::default(); MAX_CASCADES_PER_LIGHT],
+            // TODO: Is the below comment still valid?
```

Comment:
Yes. It is saying that we technically shouldn't multiply the alpha by the intensity, but because it isn't used, it doesn't matter.

Reply:
That said, we could make it a vec3 instead and move it down so one of the u32 / f32 that come after can be packed. Right now it doesn't make the struct smaller, but it does create space.

Reply:
It used to literately multiply by a variable called intensity. Idk if there's a more accurate comment now.